src/DashboardService.Business/ChangeLog/Interfaces/IGetChangeLogCommand.cs

```diff
@@ -0,0 +1,14 @@
+﻿using LT.DigitalOffice.Kernel.Responses;
+using System.Threading.Tasks;
+using LT.DigitalOffice.DashboardService.Models.Dto.Responses;
+using LT.DigitalOffice.DashboardService.Models.Dto.Requests.ChangeLog.Filter;
+using System;
+using LT.DigitalOffice.Kernel.Attributes;
+
+namespace LT.DigitalOffice.DashboardService.Business.ChangeLog.Interfaces;
+
+[AutoInject]
+public interface IGetChangeLogCommand
+{
+  public Task<OperationResultResponse<ChangeLogResponse>> ExecuteAsync(Guid id, GetChangeLogFilter filter);
```

Comment:
Why do you use here 'public' and in other interfaces?

Reply:
It is used in the controller